spec.html

```diff
@@ -12959,12 +12963,11 @@ <h1>Runtime Semantics: GetSuperConstructor ( )</h1>
 
       <!-- es6num="12.3.5.3" -->
       <emu-clause id="sec-makesuperpropertyreference" aoid="MakeSuperPropertyReference">
-        <h1>Runtime Semantics: MakeSuperPropertyReference ( _propertyKey_, _strict_ )</h1>
-        <p>The abstract operation MakeSuperPropertyReference with arguments _propertyKey_ and _strict_ performs the following steps:</p>
+        <h1>Runtime Semantics: MakeSuperPropertyReference ( _actualThis_, _propertyKey_, _strict_ )</h1>
+        <p>The abstract operation MakeSuperPropertyReference with arguments _actualThis_, _propertyKey_, and _strict_ performs the following steps:</p>
         <emu-alg>
           1. Let _env_ be GetThisEnvironment( ).
           1. Assert: _env_.HasSuperBinding() is *true*.
-          1. Let _actualThis_ be ? _env_.GetThisBinding().
```

Comment:
Should there be an assertion here about the value, or is that unnecessary?

Reply:
Unnecessary, these are the only callers. Additionally, `getThisBinding` itself has an assertion.

Reply:
I was mainly thinking to ensure future callers use it properly, but yeah you’re probably right.